_component_design/switches.md

```diff
@@ -19,29 +18,81 @@ helix-ui-javascript: false
 
 <section class="static-section" markdown="1">
 
-## Introduction
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## When to use
+
+Use a switch in the following situations:
+
+- When you want to enable to user to make a binary decision, such as turning a setting on or off. A switch does not contain a neutral option. Modifying preferences and settings is a prime use case for this component.
+- Use a switch when the user needs to modify settings or enable billing and notification preferences. If your use case is outside of these three areas, consider a [radio]({{site.baseurl}}/components/radio-buttons.html) or [checkbox]({{site.baseurl}}/components/checkboxes.html).
+
+{% endcolumn %}
+
+</div>
+
+</section>
+
+<section class="static-section" markdown="1">
 
 <div class="hxRow"  markdown="1">
 
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-4-md hxSpan-4-lg" %}
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Best practices
+
+Consider the following best practices when you construct a switch:
+
+- Construct the switch so that any change to its state is immediately saved. Use a checkbox or radio if the user performs additional steps for changes to be effective. For example, use a radio or checkbox if the user must click a **Submit** or **Next** button to apply changes.
+- A switch changes state when the user clicks it. Do not construct a switch that changes state when the user *clicks and drags* the knob.
+- For clarity, consider including animation that shows the user the switch behavior.
```

Comment:
For clarity, consider including animation that shows knob changing position.

Reply:
Done.